example/block_aware/block_aware_test.go

```diff
@@ -1,57 +0,0 @@
-package main
```

Comment:
why remove this?

Reply:
its not testing anything new. used to be when BlockAware was it's own interface. Now it's just part of Application.